encrypt.go

```diff
@@ -191,6 +245,10 @@ func (e *EncryptManager) Decrypt(r io.Reader) ([]byte, error) {
 			return nil, errors.New("no gcm decryption parameters given")
```

Comment:
Not from this change, but is that `case GCM:` repeated a few lines above?

Reply:
Nice catch, it is being repeated, I have removed this as this seems like a repetition mistake.